src/app/views/auth-views/forgot-password/forgot-password.component.scss

```diff
@@ -7,6 +7,10 @@ ion-icon.header {
     font-size: 30px;
     color: white;
 }
+small{
```

Comment:
Use class font size and white color using direct css class from Common.css no need to write a separate class